pycpfcnpj/mask.py

```diff
@@ -0,0 +1,14 @@
+
+from . import cpf
+from . import cnpj
+
+
+def mask_cpf_cnpj(cpf_cnpj: str):
+    cpf_cnpj = "".join([x for x in f'{cpf_cnpj}'])
```

Comment:
I didn't get it why are you doing this.

Reply:
In some cases I get an integer with the cpf/cnpj number. That's why I do this conversion to string.

Reply:
You should raise an exception, then. See, you annotated that `cpf_cnpj` is a string, but still are doing an operation in the case that is not, you know what I mean?